models/orm_base.py

```diff
@@ -12,3 +12,8 @@ def _asdict(self):
 
     def __str__(self):
         return f"<{type(self).__name__} {self._asdict()}>"
+
+    def update(self, **kwargs):
+        for key, value in kwargs.items():
+            if hasattr(self, key):
+                setattr(self, key, value)
```

Comment:
I have no idea why there's no built-in way to just bulk-update attributes of a model...